guides/source/active_record_basics.md

```diff
@@ -162,14 +175,42 @@ CREATE TABLE products (
 );
 ```
 
-The schema above declares a table with two columns: `id` and `name`. Each row of
-this table represents a certain product with these two parameters. Thus, you
-would be able to write code like the following:
+Database tables in Rails are typically created using [Active Record Migrations](#migrations) and not raw SQL. A migration for the `products` table above can be generated like this:
+
+```bash
+$ rails generate migration CreateProducts name:string
+```
+
+and results in this:
 
 ```ruby
+# Note: the `id` column, as the primary key, is autometically created by convension. As are `created_at` and `updated_at` columns.
+
+# /db/migrate/20240220143807_create_products.rb
+class CreateProducts < ActiveRecord::Migration
+  def change
+    create_table :products do |t|
+      t.string :name
+
+      t.timestamps
+    end
+  end
+end
+```
+
+The SQL as well as the migration above declare a table with two columns: `id`
+and `name`. Each row of this table can be represented by an instance of the
+`Product` class, with the same two attribures, `id` and `name`. You can access a
+product's attributes like this:
+
+```irb
 p = Product.new
```

Comment:
Should we add the `irb` command prompt like in other places?

```suggestion
irb> p = Product.new
```

Reply:
Oh right! so I thought about this and I left that out on purpose because it isn't actually `irb`. These examples are from a `rails console` session. In fact, if a reader starts a `irb` REPL session and tries to copy/paste, most of these code snippets won't work. They are rails application specific and not plain ruby.

I was struggling with this so ended up leaving it out. But I know the guidelines say to use `>irb` some place. I can add it back in though wondering if anyone else shares by concern about it being misleading to the reader? cc: @carlosantoniodasilva

Reply:
I couldn't find mention of this in the guides guidelines, only in the [api guidelines](https://guides.rubyonrails.org/api_documentation_guidelines.html#irb), but it's more about the output than the code.  We do use `irb>` in most Ruby code examples, so maybe for now we stick with it. The copy button does not copy that, so it's helpful there.

That said, I wonder if we should just use `>>` as it'd be what shows up in irb/rails console if not configured with a custom prompt. Perhaps that should be a discussion we could open in Basecamp, to see if it's worth changing across the board.

Reply:
Apparently the [getting started guide shows the bin/rails console opens up irb](https://edgeguides.rubyonrails.org/getting_started.html#using-a-model-to-interact-with-the-database), and shows the next examples with `irb>`, so let's keep using that.